CONTRIBUTING.md

```diff
@@ -1,20 +1,28 @@
 # Contributing
 
 - Creating a bundle: `npm prepublish`
-- Working on docs: `npm run docs:watch`
-- Developing with tests: `npm run test:watch`
+- Working on docs: `npm run docs:watch` / `yarn docs:watch`
+- Developing with tests: `npm run test:watch` / `yarn test:watch`
+- Fixing js and ts code style: `npm run style:fix` / `yarn style:fix`
```

Comment:
maybe, maybe not. let's keep it like is here with semistandard. we can iterate on next PR. There is also very powerful cross language Clang formatter to consider